asyncpg/protocol/record/recordobj.c

```diff
@@ -57,7 +57,11 @@ ApgRecord_New(PyTypeObject *type, PyObject *desc, Py_ssize_t size)
             return PyErr_NoMemory();
         }
         o = (ApgRecordObject *)type->tp_alloc(type, size);
+#ifdef _PyObject_GC_IS_TRACKED
```

Comment:
I'd define our own macro like "APG_GC_TRACKED" conditionally, so that the C code reads clearer with a single `if (!APG_GC_TRACKED(...)) {...}`